voluncheer/profiles/urls.py

```diff
@@ -1,14 +1,27 @@
 from django.urls import path
-from django.urls import re_path
 
 from profiles.views.activate_email import activate
+from profiles.views.home import SignUpView
 from profiles.views.home import home
+from profiles.views.organizations import OrganizationSignUpView
 from profiles.views.profile import ProfileView
 from profiles.views.profile import profile_update
+from profiles.views.volunteers import VolunteerSignUpView
 
 urlpatterns = [
     path("", home, name="home"),
-    re_path(r"^profile/$", ProfileView.as_view(), name="profile"),
-    re_path(r"^profile/update/$", profile_update, name="profile_update"),
```

Comment:
I added that because there's a domain issue when I update profile infos. Can you try testing our functions and see if they go well?

Reply:
I tested both volunteer and organization updating and both worked well.

My understanding of `^profile/$` is that it would match any strings that both start and end with "profile/", so wouldn't that just be "profile/"? Or "profile/profile/profile/"?

Reply:
The `^profile/$` regular expression basically says the the path must be exactly `profile/`, one a single occurrence of the content. That said, I would not expect the regular expression to be needed here.